workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/PropertyDetails/UnityCommandDetails.cs

```diff
@@ -0,0 +1,23 @@
+using Improbable.Gdk.CodeGeneration.Utils;
+
+namespace Improbable.Gdk.CodeGeneration.Model.Details
+{
+    public class UnityCommandDetails : Details
+    {
+        public new string Name => PascalCaseName;
```

Comment:
I don't think changing the meaning of a field on a derived class is a good idea. 

```csharp
public static void Foo(UnityCommandDetails details)
{
	Console.WriteLine(details.Name);
}

public static void Bar(Details details)
{
	Console.WriteLine(details.Name);
}
```

Those two functions have different behaviour for the same object (which is very unexpected imo).

Reply:
added an option to `Details` constructor to specify which should be the default case used for `Name`

Reply:
I don't think that's much better? The field `Name` should always mean the same thing. Why does it need to be pascal case or camel case if those are both available on `Details` already?